content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,116 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Improved** performance of `ServiceEntry` updates #19305.
+- **Improved**fix readiness probe inconsistency #18164.
+- **Improved** performance of configuration updates by sending partial updates where possible #18354.
+- **Added** an option to control locality load balancer settings per host #18406.
+- **Fixed** an issue where pods crashing would trigger excessive configuration pushes #18574.
+- **Fixed** issues with applications that call themselves #19308.
+- **Added** detection of `iptables` failure when using Istio CNI #19534.
+- **Added** `consecutive_5xx` and `gateway_errors` as outlier detection options #19771.
+- **Improved** `EnvoyFilter` matching performance optimization #19786.
+- **Added** support for `HTTP_PROXY` protocol #19919.
+- **Improved** `iptables` setup to use `iptables-restore` by default #18847.
+- **Enabled** [automatic protocol detection](https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection-experimental) by default #18666.
+
```

Comment:
The following were listed under configuration. Should they belong here instead?

```
- Implement extra environment variables from `ProxyMetadata` #20447.
- Make headless service passthrough resolution optional #19992.
```

Reply:
@howardjohn any opinion on this?

Reply:
can we add some context - what does proxy metadata do and why i need to care about it?  same for headless service passthrough resolution.